### PR TITLE
refactored diff as a standalone tool that checks the databases (and o…

### PR DESCRIPTION
**Pull request by vidiecan**
…nly them)

| Phases            | MP | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|----:|-----:|-----:|-------:|
| ETA                  |  0  |  0  |    0 |     0 |      0 |        0 |
| Developing      |  0  |  0  |    0 |    0 |      1 |         1 |
| Review             |  0  |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -  |   -   |  -    |   -    |         1 |
| ETA est.             |      |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -  |   -   |   -     |        0 |

Supersedes:
* https://github.com/dataquest-dev/dspace-import/pull/149
* https://github.com/dataquest-dev/dspace-import/pull/146

### CODE REVIEW

File: tools/repo_diff/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Import check
+
+1. Get database dumps and unzip them into `input/dump`, rename to `dspace6.sql` and `dspace7.sql` accordingly.
+ 
+1. Create `dspace6` and `dspace7` databases from dump:
```
Comment (Paurikova2) [inline tools/repo_diff/README.md:5]: Step 2. not 1.

File: tools/repo_diff/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+```
+> scripts/start.dspace.db.bat
+Starting postgres
+Importing dspace6
+Importing dspace7
+Done, starting psql
+Waiting for PID:7 /usr/local/bin/docker-entrypoint.sh
+```
+
+1. Add settings to `diff_settings`, see [settings](diff_settings/tul.py).
```
Comment (Paurikova2) [inline tools/repo_diff/README.md:15]: Step 3. not 1.

File: tools/repo_diff/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+Starting postgres
+Importing dspace6
+Importing dspace7
+Done, starting psql
+Waiting for PID:7 /usr/local/bin/docker-entrypoint.sh
+```
+
+1. Add settings to `diff_settings`, see [settings](diff_settings/tul.py). 
+
+1. Compare dspace databases:
```
Comment (Paurikova2) [inline tools/repo_diff/README.md:17]: Step 4. not 1.

**Review by Paurikova2 — CHANGES_REQUESTED** [review]

File: tools/repo_diff/README.md (hunk 1 above)
Comment (vidiecan) [reply tools/repo_diff/README.md:5]: these are automatically incremented
https://www.markdownguide.org/basic-syntax/#ordered-lists

**Review by Paurikova2 — APPROVED** [review]

**Pull request merged**